Formula/bork.rb

```diff
@@ -4,7 +4,7 @@ class Bork < Formula
   url "https://github.com/skylarmacdonald/bork/archive/v0.12.0.tar.gz"
   sha256 "525f797a5ad01734d298852a038c2a2cb338ee9bb038c560bb20ecf142b1588b"
   license "Apache-2.0"
-  head "https://github.com/skylarmacdonald/bork.git"
+  head "https://github.com/skylarmacdonald/bork.git", branch: "main"
```

Comment:
https://docs.brew.sh/Formula-Cookbook#unstable-versions-head

Docs say the syntax is:

```
class Foo < Formula
  head "https://github.com/some/package.git", :revision => "090930930295adslfknsdfsdaffnasd13"
                                         # or :branch => "develop" (the default is "master")
                                         # or :tag => "1_0_release",
                                         #    :revision => "090930930295adslfknsdfsdaffnasd13"
end
```

But that syntax causes a `brew audit --strict bork` to fail, and looking in `Formula`, seems like nothing uses the syntax listed in the docs.

Hence: https://github.com/Homebrew/homebrew-core/pull/76462/commits/9200bd0b877ef48ee141e08e87afedf685097185

Reply:
Oops, thanks for pointing this out. I've [put up a PR](https://github.com/Homebrew/brew/pull/11303) to fix the example!